ext/standard/formatted_print.c

```diff
@@ -644,7 +644,16 @@ php_formatted_print(zend_execute_data *execute_data, int use_array, int format_o
 			inpos++;
 		}
 	}
-
+	
+	if (argc - 1 > argnum) {
+		efree(result);
+		if (newargs) {
+			efree(newargs);
+		}
+		php_error_docref(NULL, E_WARNING, "Too much arguments");
+		return NULL;
```

Comment:
The warning is a BC, and returning false is a much larger one, so I'd like to discuss if we should rather just add the warning, but and keeping returning the new string.

Reply:
Why the `return NULL` though? You have all the information you need to process it correctly, so the warning is fine, but you should still let it return the result.